settings.xml

```diff
@@ -1,93 +1,17 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      http://maven.apache.org/xsd/settings-1.0.0.xsd">
+
   <servers>
     <server>
-      <username>${artifactory-username}</username>
-      <password>${artifactory-password}</password>
-      <id>artifactory</id>
-    </server>
-    <server>
-      <username>${artifactory-username}</username>
-      <password>${artifactory-password}</password>
-      <id>the-lucky-gunslingers</id>
-    </server>
-    <server>
-      <username>${artifactory-username}</username>
-      <password>${artifactory-password}</password>
-      <id>snapshots</id>
+      <id>zextras-public</id>
+      <username>${iam-user-access-key-id}</username>
+      <password>${iam-user-secret-key}</password>
+      <configuration>
+        <region>eu-west-1</region>
+        <publicRepository>true</publicRepository>
+      </configuration>
     </server>
```

Comment:
You need to add at least `the-lucky-gunslingers` and `zextras-java-sdk` servers credentials, otherwise not all dep will be resolved by maven

```suggestion
    </server>
    <server>
      <id>the-lucky-gunslingers</id>
      <username>${artifactory-username}</username>
      <password>${artifactory-password}</password>
    </server>
    <server>
      <id>zextras-java-sdk</id>
      <username>${artifactory-username}</username>
      <password>${artifactory-password}</password>
    </server>
```